static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -393,20 +395,23 @@ class GroupDetails extends Component<Props, State> {
     GroupStore.onPopulateReleases(this.props.params.groupId, releases);
   }
 
-  async fetchReplaysCount() {
+  async fetchReplayIds() {
     const {api, location, organization, params} = this.props;
     const {groupId} = params;
 
-    this.setState({loadingReplaysCount: true});
+    this.setState({loadingReplayIds: true});
 
-    const eventView = EventView.fromSavedQuery({
-      id: '',
-      name: `Replays in issue ${groupId}`,
-      version: 2,
-      fields: ['count()'],
-      query: `issue.id:${groupId}`,
-      projects: [],
-    });
+    const eventView = EventView.fromNewQueryWithLocation(
+      {
+        id: '',
+        name: `Errors within replay`,
+        version: 2,
+        fields: ['id', 'replayId'],
```

Comment:
Do you need `id` here?

Reply:
`id` seems like it's included no matter what, i thought i would type it out to be explicit.

Reply:
ah, this shouldn't be listed there when the `count()` aggregate is used. Coming out now!